pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -193,199 +221,87 @@ def get_commuter_data(read_data=dd.defaultDict['read_data'],
     max_abs_err = 0
```

Comment:
Can you fix the typo in line 220?
   maxium errors (of people not detected) ->    maximum errors (of people not detected)